web/blog/tenzir-v4.4/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Tenzir v4.4
+authors: [dominiklohmann]
+date: 2023-11-03
```

Comment:
```suggestion
date: 2023-11-06
```

We moved this to Monday just now, there was little time today to cut the release.